kubernetes/apps/kube-system/cilium/gateway/certificates/lab-casa.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: laboratory-casa-gateway-production
+spec:
+  secretName: laboratory-casa-gateway-production-tls
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  commonName: laboratory.casa
+  dnsNames:
+    - laboratory.casa
+    - "*.laboratory.casa"
```

Comment:
I don't think I actually need a new SSL cert, but it's one more easy way when testing to make sure my browser is hitting the new gateway and not using any cached stuff. Plus the certs currently live in the [`ingress-nginx` app](https://github.com/mebezac/home-cluster/blob/main/kubernetes/apps/network/ingress-nginx/certificates/secret-generator.yaml?plain=1#L10-L15), so they'll need to move eventually anyways.